styles/components/button.scss

```diff
@@ -59,41 +43,63 @@
 
   transition: 0.1s all;
 
-  &:hover {
```

Comment:
How about the text-decoration. It definitely doesn't belong to `text-decoration`.